src/Services/ApmCollectorService.php

```diff
@@ -44,10 +44,10 @@ public function __construct(Application $app, Dispatcher $events, Config $config
     public function startMeasure(
         string $name,
         string $type = 'request',
-        ?string $action = null,
-        ?string $label = null,
-        ?float $start_time = null
```

Comment:
since this is a public function's declaration change it could potentially break compatibility for somebody that was passing `null` here. Unlikely, but just mentioning.

Reply:
Why did the linter remove the `?` then? Do you think we should override the rule?